aptos-move/framework/aptos-stdlib/sources/string_utils.move

```diff
@@ -53,31 +54,31 @@ module aptos_std::string_utils {
     fun test_format_list() {
         let my_string = std::string::utf8(b"My string");
         let l = cons(1, cons(2, cons(my_string, nil())));
-        let s = format_list(&std::string::utf8(b"a = {} b = {} c = {}"), &l);
+        let s = format_list(&b"a = {} b = {} c = {}", &l);
         assert!(s == std::string::utf8(b"a = 1 b = 2 c = \"My string\""), 1);
     }
 
     #[test]
     #[expected_failure(abort_code = EARGS_MISMATCH)]
     fun test_format_list_to_many_vals() {
         let l = cons(1, cons(2, cons(3, cons(4, nil()))));
```

Comment:
Now we would need an infix operator, than this could be even readable, e.g.: 

```
format(&b"{}/{}" , nominator<<denominator)
```

I definitely would like to have them in the language, and hopefully we have them in a quarter or two.

Reply:
This could technically be done currently as well. We can support something like

format(&b"{}/{}", vector[nominator, denominator]).

Would only work for values of the same type, but that should work for most common cases I think